pkg/sensors/handler.go

```diff
@@ -94,10 +94,19 @@ func (h *handler) updatePolicyFilter(tp tracingpolicy.TracingPolicy, tpID uint64
 }
 
 func (h *handler) addTracingPolicy(op *tracingPolicyAdd) error {
+
+	var namespace string
+	if tpNs, ok := op.tp.(tracingpolicy.TracingPolicyNamespaced); ok {
+		namespace = tpNs.TpNamespace()
+	} else {
+		namespace = ""
+	}
```

Comment:
```suggestion
	}
```
nit: `else` is not needed here

Reply:
Updated this as well.